### PR TITLE
Consolidate processors: Nifi LDES version materialisation processor.

### PR DESCRIPTION
**Pull request by sandervd**

### CODE REVIEW

File: nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            .Builder().name("MembersOnly")
+            .displayName("Restrict output to members")
+            .description("When enabled, only the member and the blank nodes references are included.")
+            .required(false)
+            .defaultValue("false")
+            .allowableValues("true", "false")
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .build();
+
+
```
Comment (ddvlanck) [inline nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java:70]: Remove this line

**Review by ddvlanck — COMMENTED** [review]
/

File: nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        }
+        AtomicBoolean is_successful = new AtomicBoolean(false);
+        StringWriter outputStream = new StringWriter();
+        session.read(flowFile, new InputStreamCallback() {
+            @Override
+            public void process(InputStream in) throws IOException {
+                try {
+                    String FragmentRDF = IOUtils.toString(in);
+                    InputStream targetStream = IOUtils.toInputStream(FragmentRDF);
+                    Model inputModel = Rio.parse(targetStream, "", RDFFormat.NQUADS);
```
[the remark this answers is not in the document]
Comment (sandervd) [reply nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java]: Nice catch, was indeed a leftover from an earlier version...

File: nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java (hunk 1 above)
Comment (sandervd) [reply nifi-ldes-version-materialisation-processor/src/main/java/be/vlaanderen/informatievlaanderen/processors/VersionMaterialiseProcessor.java]: Removed

**Review by ddvlanck — APPROVED** [review]

**Pull request merged**